SparkPost/matchbox
# UX-436 Resolve dependabot vulnerabilities

### What Changed
- Resolves:
  - node-notifier: https://github.com/SparkPost/matchbox/security/dependabot/package-lock.json/node-notifier/open 
  - highlight.js: https://github.com/SparkPost/matchbox/security/dependabot/packages/design-tokens/package-lock.json/highlight.js/open
- Does NOT resolve (unable to update since parent dependencies haven't patched them yet):
  - ecstatic: https://github.com/SparkPost/matchbox/security/dependabot/package-lock.json/ecstatic/open
  - immer: https://github.com/SparkPost/matchbox/security/dependabot/package-lock.json/immer/open
  - socket.io: https://github.com/SparkPost/matchbox/security/dependabot/site/package-lock.json/socket.io/open



### How To Test or Verify
- Verify the package-lock changes are sufficient to resolve the dependabot alerts
- Verify tests and builds work


### PR Checklist
